setup.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -e
+
+# check os
+uname_out=$(uname -s)
+case ${uname_out} in
+    Linux)      installer='sudo apt install' ;;
+    Darwin)     installer='brew install' ;;
+    *)          echo 'Your system is not supported, choose installer manually.'
+                exit 1
+esac
+
+${installer} python3.6
+${installer} python3.6-venv
+
+# set up virtualenv
+PROJECT_PATH=$(dirname $0)
+python3.6 -m venv ${PROJECT_PATH}/venv
```

Comment:
You are using python 3.6 but as far as i know (official doc) naoqi works only with 2.7, is it tested?

Reply:
It works because I don't use naoqi anywhere. I just publish commands using paho mqqt. If you think naoqi will be needed I can change to python2.7

Reply:
I'm pretty sure that we need to use naoqi

Reply:
done